src/tiles/list/tile-list-compact.ts

```diff
@@ -0,0 +1,204 @@
+import { css, html, LitElement, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
+import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import { accountLabel } from './account-label';
+import '../../mediatype-icon';
+
+@customElement('tile-list-compact')
+export class TileListCompact extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam: SortParam | null = null;
+
+  @property({ type: Number }) mobileBreakpoint?: number;
+
+  render() {
+    return html`
+      <div id="list-line" class="${this.classSize}">
+        <div id="thumb">${this.imageTemplate}</div>
+        <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+        <div id="creator">
+          ${this.model?.mediatype === 'account'
+            ? accountLabel(this.model?.dateAdded)
+            : DOMPurify.sanitize(this.model?.creator ?? '')}
+        </div>
+        <div id="date">${formatDate(this.date, this.formatSize)}</div>
+        <div id="icon">
+          <mediatype-icon .mediatype=${this.model?.mediatype}> </mediatype-icon>
```

Comment:
QUESTION: Intentional space between the opening and closing tag?

Reply:
I'll stop commenting on whitespace. Suffice to say, I'm not sure if there's a reason for whitespace being added at render time before custom tags, after custom tags, or in between the open & close tags for a custom tag. If not intentional, then there are a handful of places that could be cleaned up.

Reply:
I don't know about all of the spots, but a lot of the formatting is auto-formatted by `prettier` so it's possible there are rules in there for the whitespace like this. We have a `git` pre-commit hook that auto-formats the code before it's committed to the repo.